resources/sts/4.12/shared_vpc_openshift_ingress_operator_cloud_credentials_policy.json

```diff
@@ -0,0 +1,20 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "elasticloadbalancing:DescribeLoadBalancers",
+                "route53:ListHostedZones",
+                "route53:ChangeResourceRecordSets",
+                "tag:GetResources"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
```

Comment:
Please add condition key here restricting the domains we have the ability to modify records for.

Reply:
Added the condition key to `ChangeResourceRecordSets` to match the managed policy we have for classic ROSA.